_layouts/sidenav.html

```diff
@@ -0,0 +1,40 @@
+---
+layout: main
+---
+
+{% capture sidenav %}
+  <nav aria-label="{% t accessible_labels.secondary_navigation %}">
+    <ul class="usa-sidenav">
+      <li class="usa-sidenav__item">
+        <a href="/contact/" class="usa-current">{% t page.title %}</a>
+        <ul class="usa-sidenav__sublist">
+          {% for item in page.sidenav %}
+          <li class="usa-sidenav__item">
+            <a href="#{{ item.url }}">{{ item.name }}</a>
```

Comment:
Would it be correct to assume we could want to use this layout for sections like "Privacy & security" where each of the sub-items would be standalone pages, not fragments within the current page. In which case, it may be better to assume `url` is a fully-formed URL, then in the instance here of `accessibility.md` pass those as `#our-commitment`, `#standards`, `#evaluation`.

```suggestion
            <a href="{{ item.url }}">{{ item.name }}</a>
```